azerothcore/azerothcore-wotlk
# feat(DB/lang): Import german quest_request_items texts from GTDB (Part 2/4)

Part 2/4 to import the german quest_request_items texts from GTDB.
None of these IDs previously existed in German, so they were displayed in English.





## CHANGES PROPOSED:
-  Import the second part ~ 750 of a total of ~ 3000 German quest texts for table `quest_request_items_locale` from GTDB 

## TESTS PERFORMED:
- Corrected each string with the punctuation marks from the `quest_request_items` table
- Checked for spelling and grammatical errors and corrected
- Imported on win 10 pro x64 with HeidiSQL without errors
- Checked a few quests ingame




## HOW TO TEST THE CHANGES:

To test changes ingame you need deDE client. Since I read almost every string and compared the correctness using the `quest_request_items` table, it is sufficient to just test the SQL. 

- Import SQL script and check that no error occurs
- Can be merged after approval. 

## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
